server/src/test/java/org/cloudfoundry/identity/uaa/provider/oauth/ExternalOAuthLogoutHandlerTest.java

```diff
@@ -134,4 +134,15 @@ void getOAuthProviderForAuthentication() {
   void getNullOAuthProviderForAuthentication() {
     assertEquals(null, oAuthLogoutHandler.getOAuthProviderForAuthentication(null));
   }
+
+  @Test
+  void getPerformRpInitiatedLogout() {
+    oAuthIdentityProviderDefinition.setPerformRpInitiatedLogout(true);
+    assertEquals(true, oAuthLogoutHandler.getPerformRpInitiatedLogout(oAuthIdentityProviderDefinition));
```

Comment:
Yes, also `assertNull`. Intellij highlights all these opportunities for simplification for us. My thought was that I'm gonna following exactly the existing pattern, and then issue a separate PR later for refactors (simplifying not just the tests added here but also the other tests), so that this PR is minimal & the code pattern is consistent within the same class.